### DISCUSSION TITLE
App: Config limit in export data

### DISCUSSION THREAD

**Discussion by diegoleme** [directus/directus#10360] [Archive]
![image](https://user-images.githubusercontent.com/374857/144961980-7fa703ce-b74e-47df-9c58-3932bc2a7d66.png)

Always export with default of 100 items.

**Comment by azrikahar** [comment]
Duplicate of #4248.

Currently the App does indeed have this limit (technically the API), but you can circumvent it by exporting via the API directly, as mentioned here: https://github.com/directus/directus/discussions/4248#discussioncomment-1568518

Feel free to add further comments on that discussion so that we can consolidate things easier 😊
